### PR TITLE
Pp info notification

### PR DESCRIPTION
**Pull request by sktoiva**

### CODE REVIEW

File: app/view_utils/paypal_helper.rb [hunk 1]
```diff
@@ -20,4 +20,8 @@ def user_and_community_ready_for_payments?(user, community)
     PaypalHelper.personal_account_prepared?(user, community) &&
       PaypalHelper.community_ready_for_payments?(community)
   end
+
+  def missing_payment_info?(user, community)
```
Comment (ovan) [inline app/view_utils/paypal_helper.rb]: Naming should make it clear that the result is dependent on user having open listings, not just missing payment info.

File: app/view_utils/paypal_helper.rb [hunk 2]
```diff
@@ -20,4 +20,8 @@ def user_and_community_ready_for_payments?(user, community)
...
+    community.paypal_enabled && user.listings.present? && !user_and_community_ready_for_payments?(user, community)
```
Comment (ovan) [inline app/view_utils/paypal_helper.rb]: Should this only check for open listings, not closed?

**Comment by ovan** [comment]
Can you rebase on top of paypal-payment-settings? This will only be used after we get that merged and it changes how payment settings for community are stored in case of PayPal.

File: app/view_utils/paypal_helper.rb (hunk 2 above)
Comment (sktoiva) [inline app/view_utils/paypal_helper.rb]: Well, maybe. Don't know if it makes that much difference - but might give a better impression when testing.

**Comment by sktoiva** [comment]
Checked, a really good move to right direction.

**Pull request merged**